lib/api/src/getSize.js

```diff
@@ -10,7 +10,7 @@ export default function (path, cb) {
     } else {
       let fileLength = 0
       for (let i = 0; i < file.EndofFile.length; i++) {
-        fileLength |= file.EndofFile[i] << (i * 8)
+        fileLength += file.EndofFile[i] * Math.pow(2, i * 8);
```

Comment:
This is not true for the upstream code from https://github.com/bchelli/node-smb2, but for the newest content in lib/api/src (es6/es7 meant to be transpiled).